CHANGELOG.md

```diff
@@ -55,6 +55,7 @@
 * [FEATURE] Add OpenStack Swift support to blocks storage. #3303
 * [FEATURE] Added support for applying Prometheus relabel configs on series received by the distributor. A `metric_relabel_configs` field has been added to the per-tenant limits configuration. #3329
 * [FEATURE] Support for Cassandra client SSL certificates. #3384
+* [FEATURE] Add support for zone aware reads. #3414
```

Comment:
:wave: The v1.5 release just started and you will need to rebase to the latest master and move any CHANGELOG entries into the unreleased section. Thanks!

Reply:
Thanks for letting me know. A rebase has been done.